plugins/org.jkiss.dbeaver.ext.generic/plugin.xml

```diff
@@ -552,6 +552,7 @@
                     <parameter name="supports-indexes" value="true"/>
                     <parameter name="supports-stored-code" value="false"/>
                     <parameter name="supports-truncate" value="false"/>
```

Comment:
Doesn't look like a good idea. We have many SQL Server drivers, we don't want't to copy-paste it everywhere.
Let's use Java level function for that (we already have one in SQLDialect)